### PR TITLE
chore: fix all `@typescript-eslint/no-non-null-assertion` warnings

### PR DESCRIPTION
**Pull request by gitstart-calcom**
fix all @typescript-eslint/no-non-null-assertion warnings

### CODE REVIEW

**Review by hariombalhara — DISMISSED** [review]
Left a comment, apart from that it's good. Thanks @gitstart-calcom

**Comment by gitstart-calcom** [comment]
Thank you for the review @hariombalhara. PR has been updated, please re-review

**Review by PeerRich — DISMISSED** [review]

File: apps/web/components/getting-started/steps-views/SetupAvailability.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ const SetupAvailability = (props: ISetupAvailabilityProps) => {
   const { nextStep } = props;
 
   const queryAvailability = trpc.viewer.availability.schedule.get.useQuery(
-    { scheduleId: defaultScheduleId! },
+    { scheduleId: defaultScheduleId ?? undefined },
```
Comment (keithwillcode) [inline apps/web/components/getting-started/steps-views/SetupAvailability.tsx:24]: @sean-brydon Is this what we want? Or should this throw an error if defaultScheduleId is null/undefined?

**Review by keithwillcode — CHANGES_REQUESTED** [review]
Blocking to get @sean-brydon feedback before merging.

File: apps/web/components/getting-started/steps-views/SetupAvailability.tsx (hunk 1 above)
Comment (keithwillcode) [reply apps/web/components/getting-started/steps-views/SetupAvailability.tsx:24]: Talked to Sean. All good.

**Review by keithwillcode — DISMISSED** [review]

**Review by keithwillcode — APPROVED** [review]

**Pull request merged**